### DISCUSSION TITLE
Private repos not showing up in Workspaces

### DISCUSSION THREAD

**Discussion by jamesbraza** [dosu-ai/community#3] [Q&A] [answered]
I have installed the GitHub App for Dosu into a few private repos.

![screenshot of DosuBot installed app](https://github.com/dosu-ai/community/assets/8990777/a0faaadb-a2e6-4fa4-a59a-6c5407d828d3)

Then, when I go to make a workspace at https://app.dosu.dev/workspaces/create, I see:

> There are no repositories available.

![screenshot of Workspace creation](https://github.com/dosu-ai/community/assets/8990777/0e574c29-c188-4237-a29d-871d0e5ac0d1)

Do I need to do some additional configuration for private repos?

**Comment by devstein** [comment, marked as answer]
Resolved in [Slack](https://dosucommunity.slack.com/archives/C06T4PSQ7V4/p1720629620615879)

Due to a bug where users can only see repositories if they are the GitHub app installer
